Makefile

```
@@ -48,24 +48,24 @@ clean-test:
 	rm -fr .cache/
 	rm -f .coverage
 	rm -fr htmlcov/
-	rm -f test-results.xml
+	rm -fr test-reports/
 
 lint:
-	pre-commit run --all-files --show-diff-on-failure
+	poetry run pre-commit run --all-files
 
 test:
-	pytest
+	poetry run pytest
 
 test-all:
-	tox --parallel=auto
+	poetry run tox
 
 test-integration: install
-	behave integration-tests/
+	poetry run behave integration-tests/
 
 docs:
 	rm -f docs/sceptre.rst
 	rm -f docs/modules.rst
-	sphinx-apidoc -o docs/ sceptre
+	poetry run make docs
```

I think this was a whoopsie. Running `make docs` infinitely calls `poetry run make docs`. 
```suggestion
	poetry run sphinx-apidoc -o docs/ sceptre
```